core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_radiobutton.scss

```diff
@@ -1,68 +1,75 @@
 @use "variables" as *;
 
-.p-radiobutton {
-    width: 20px;
-    height: 20px;
-    border-radius: 50%;
-    transition: box-shadow 0.2s;
-}
+p-radiobutton.p-element {
+    gap: $spacing-1;
 
-.p-radiobutton .p-radiobutton-box {
-    border: 2px solid $color-palette-gray-700;
-    background: $white;
-    width: 20px;
-    height: 20px;
-    color: $black;
-    border-radius: 50%;
-    transition: background-color 0.2s, border-color 0.2s, color 0.2s, box-shadow 0.2s,
-        background-size 0.2s cubic-bezier(0.64, 0.09, 0.08, 1);
-}
+    &.ng-dirty.ng-invalid {
+        .p-radiobutton:not(.p-radiobutton-disabled) {
+            border-color: $color-alert-red;
+        }
+    }
 
-.p-radiobutton .p-radiobutton-box:not(.p-disabled).p-focus {
-    outline: 0 none;
-    outline-offset: 0;
-    box-shadow: none;
-    border: 2px solid $border-color-highlight;
-}
+    .p-radiobutton {
+        height: 24px;
+        aspect-ratio: 1/1;
+        border-radius: 50%;
+        border: 2px solid $color-palette-gray-500;
 
-.p-radiobutton .p-radiobutton-box .p-radiobutton-icon {
-    width: 10px;
-    height: 10px;
-    transition-duration: 0.2s;
-    background-color: $color-palette-primary;
-}
+        .p-radiobutton-box {
+            height: 20px;
+            border-radius: inherit;
+            aspect-ratio: inherit;
 
-.p-radiobutton .p-radiobutton-box.p-highlight {
-    border-color: $color-palette-primary;
-    background: $white;
-}
+            .p-radiobutton-icon {
+                height: 14px;
```

Comment:
Do we want the icon height to be fixed?

Reply:
As far as I know. For Icons it's good to have pixels but for the rest I'm not sure if we should use rem values